kano_avatar/logic.py

```diff
@@ -668,6 +674,8 @@ def get_avatar_conf(aux_files=[]):
         content_dir.register_path(
```

Comment:
Do you mean, we will need to change something I have written here in kano-profile?